app/controllers/application_controller.rb

```diff
@@ -2,4 +2,24 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  before_filter :api_authentication
```

Comment:
Yes i normally do that way for rails api apps by creating separate `API::Controller`
But it was just one api request requirement so i done this way